server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -80,6 +80,18 @@ def on_update(self, updates, original):
         self._check_equals([doc])
         self._check_parameters([doc])
 
+    def delete(self, lookup):
+        referenced_filters = self._get_referenced_filter_conditions(lookup.get('_id'))
+        if referenced_filters.count() > 0:
+            references = ''.join([pf['name'] for pf in referenced_filters])
```

Comment:
It's better to join the publish filter names with a separator when we are expecting to have more than one in Mongo.